src/test/java/de/bonndan/nivio/appuser/AppUserRoleTest.java

```diff
@@ -0,0 +1,18 @@
+package de.bonndan.nivio.appuser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+class AppUserRoleTest {
```

Comment:
You might tests valueof(null) returning a sane default

Reply:
I do not understand 100 %, let's talk about it, tomorrow.